Core/Core/Profile/Settings/APIPairingCode.swift

```diff
@@ -31,6 +31,19 @@ public struct APIPairingCode: Codable, Equatable {
     let workflow_state: String?
 }
 
+public struct GetAccountTermsOfService: APIRequestable {
```

Comment:
We typically end the requestable name with `Request` so this would be named `GetAccountTermsOfServiceRequest` and the UseCase would be called `GetAccountTermsOfService`.

Reply:
Doh